valor-software/ng2-bootstrap
# fix(datepicker): fixed clear button in datepicker

fix(datepicker): fixed clear button in datepicker
close #6335 #6336 

What to test:
In datepicker was fixed clear button.

Preconditions:
use preview channel from this pr

How to test:
go to datepicker page on documentation, find section with title 'Show Clear Button', choose date on datepicker and then click clear button. The choose date should be cleaned.
